sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/uninstall/UninstallScheduler.java

```diff
@@ -94,19 +93,28 @@ private Plan getPlan() {
 
         List<Phase> phases = new ArrayList<>();
 
+        // First, we kill all the tasks, so that we may release their reserved resources.
+        List<Step> taskKillSteps = stateStore.fetchStatuses().stream()
```

Comment:
Any reason for getting the ids from the statuses rather than the infos? Should probably have a comment explaining why